test/sql/constraints/foreignkey/test_fk_persistent_storage.test

```diff
@@ -0,0 +1,64 @@
+# name: test/sql/constraints/foreignkey/test_fk_persistent_storage.test
+# description: persistent storage tests with foreign key tables
+# group: [foreignkey]
+
+# load the DB from disk
+load __TEST_DIR__/test_fk_persistent.db
```

Comment:
Can we move this test to the `storage` directory?

Reply:
Okay, I will move it storage directory.